app/assets/stylesheets/src/shared/_components.scss

```diff
@@ -276,31 +273,35 @@
 // In this case we moved the dropdown styles out of the `.select-component` class.
 .selectize-dropdown {
   border-radius: 4px;
+  border: 1px solid rgba(4, 32, 69, 0.1);
```

Comment:
did not find the variable with `#042045`, [used plain rgba](https://www.color-hex.com/color/042045)

Reply:
```suggestion
  border: 1px solid var(--tblr-border-color-translucent);
```

Reply:
![image](https://github.com/user-attachments/assets/a2eee999-7088-43e5-85a2-c9272e91813c)